MSAL/test/integration/native_auth/end_to_end/sign_in/MSALNativeAuthSignInUsernameEndToEndTests.swift

```diff
@@ -118,131 +97,23 @@ final class MSALNativeAuthSignInUsernameEndToEndTests: MSALNativeAuthEndToEndBas
         let username = ProcessInfo.processInfo.environment["existingOTPUserEmail"] ?? "<existingOTPUserEmail not set>"
```

Comment:
Should this also be changed to `let username = "missing email+code signIn username"`?

Reply:
All the skipped tests need to be revisited once enabled. So, I'd say to do this once we enable them